dd-sdk-android/src/main/java/com/datadog/opentracing/propagation/DatadogHttpCodec.java

```diff
@@ -49,6 +50,9 @@ public void inject(final DDSpanContext context, final TextMapInject carrier) {
       for (final Map.Entry<String, String> entry : context.baggageItems()) {
         carrier.put(OT_BAGGAGE_PREFIX + entry.getKey(), HttpCodec.encode(entry.getValue()));
       }
+
+      // makes this request trace selected for sampling
+      carrier.put(SELECTED_FOR_SAMPLE_KEY, "1");
```

Comment:
is there a relationship between `x-datadog-sampling-priority` and `x-datadog-sampled`? If priority is say `0`, this should be still `1`?

Reply:
Is what I want to ask also during the daily.

Reply:
I didn't find values other than `1` in the DD code https://github.com/search?p=4&q=org%3ADataDog+%22x-datadog-sampled%22&type=Code

Anyway, iOS SDK is using `1` https://github.com/DataDog/dd-sdk-ios/blob/887b1429b8ec3804b194783aa423f989fcaac06d/Sources/Datadog/Tracing/Propagation/TracingHTTPHeaders.swift#L27

But at the same time on iOS `x-datadog-sampling-priority` is always `1`, unlike on Android.

Reply:
Out of consistency with iOS and React Native, let's make sure the `x-datadog-sampling-priority` is also always one.

Reply:
You can read more on this in https://github.com/DataDog/dd-sdk-ios/pull/575, notably that these values were taken from [this doc](https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces/?tab=browserrum#how-are-rum-resources-linked-to-traces).